docs/getting-started/clients/browser/index.md

```diff
@@ -8,6 +8,79 @@ sidebar_position: 3
 
 Before you start, you must complete the [Clients repository setup instructions](../index.md).
 
+## Environment Setup
```

Comment:
What do you think about putting this section below the `Build Instructions` section? It is pre-configuration, but seems a little more in-depth than the run instructions and can be used after building and running if people wish to do so

Reply:
Agreed.

Reply:
Great idea!  Done.